### PR TITLE
slib: Move to library section and update to version 3b5

### PR DESCRIPTION
**Pull request by hliebel**

### CODE REVIEW

File: components/library/slib/Makefile [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+COMPONENT_PROJECT_URL=	http://people.csail.mit.edu/jaffer/SLIB/
+COMPONENT_SUMMARY=	GNU slib - a portable library for the Scheme programming language
+COMPONENT_DESCRIPTION=	SLIB is a portable scheme library providing compatibility and utility functions for standard Scheme implementations.
+COMPONENT_SRC=		$(COMPONENT_NAME)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC)-$(COMPONENT_VERSION).zip
+COMPONENT_ARCHIVE_HASH=	\
+    sha256:f8b64a9820c2ccbdb263a0cb7bcf4caadce0857fdf4e2f52111ca33b4a131760
+COMPONENT_ARCHIVE_URL=	http://groups.csail.mit.edu/mac/ftpdir/scm/$(COMPONENT_ARCHIVE)
+COMPONENT_LICENSE= 	BSD-like, PD
+COMPONENT_LICENSE_FILE= $(COMPONENT_NAME).license
```
Comment (alarcher) [inline components/library/slib/Makefile]: Remove the COMPONENT_LICENSE_FILE line.

**Review by alarcher — COMMENTED** [review]
Could you list the consumers of this component?

**Comment by hliebel** [comment]
Mmm... looks like no consumers at the moment doing a 'pkg search require:'

Ah, didnt know the license file had a default macro..

File: components/library/slib/slib.p5m [hunk 2]
```diff
@@ -63,6 +57,7 @@ file path=usr/lib/slib/colornam.scm
 file path=usr/lib/slib/colorspc.scm
 file path=usr/lib/slib/comlist.scm
 file path=usr/lib/slib/comparse.scm
+file path=usr/lib/slib/configure
```
Comment (pyhalov) [inline components/library/slib/slib.p5m]: How is this file used? Should it be delivered with slib at all?

**Pull request merged**